packages/contract/lib/execute.js

```diff
@@ -525,20 +525,6 @@ const execute = {
   //(well, OK, it's lacking some things there too, but again, good enough
   //for our purposes)
   sendTransaction: async function (web3, params, promiEvent, context) {
-    // get the chainId to be compliant with EIP-155
-    // Geth wants this value in base 16 prefixed by "0x"
-    let chainId;
-    if (context.contract._chainId) {
-      chainId = context.contract._chainId;
```

Comment:
If you're removing this, you should also remove the line I added in `constructorMethods.js` to invalidate the cache.  (See #3947.)